packages/sanes-chrome-extension/src/extension/background/actions/createPersona/requestSenderWhitelist.ts

```diff
@@ -0,0 +1,64 @@
+const ALLOWED_URLS_KEY = 'auk';
+const BLOCKED_URLS_KEY = 'buk';
+const SEPARATOR = ',';
+
+export class SenderWhitelist {
```

Comment:
We have `ShareIdentity` block list and `Transaction Request` block list. Should we use one list for all purposes?

Reply:
As we have the code today we can get rid of the allowed list. We do not process any request automatically. Coded in this way because I had in mind the recurring transactions which at the end we are not going to implement.

Good catch.